scripts/publish_infra_k8s_outputs.py

```
@@ -0,0 +1,150 @@
+) -> int:
+    """Publish outputs for the wildside-infra-k8s action.
+
+    This command reads computed values from environment variables and
+    writes them to GITHUB_OUTPUT for use by subsequent workflow steps.
+    """
+    # Resolve GITHUB_OUTPUT path
+    github_output_raw = resolve_input(
+        github_output,
+        InputResolution(
+            env_key="GITHUB_OUTPUT",
+            default=Path("/tmp/github-output-undefined"),
```

Ruff (0.14.13)

Probable insecure usage of temporary file or directory: "/tmp/github-output-undefined"

(S108)